cmd/flags_misc.go

```diff
@@ -182,15 +184,17 @@ func ParseMiscFlags(cmd *cobra.Command, env *Env) error {
 
 	ok, err = cmd.Flags().GetBool(metricsFlag)
 	if err != nil {
-		return err
+		panic(err)
 	}
 
 	if ok {
 		opts := []otlpmetrichttp.Option{
 			otlpmetrichttp.WithCompression(otlpmetrichttp.GzipCompression),
 			otlpmetrichttp.WithEndpoint(cmd.Flag(tracingEndpointFlag).Value.String()),
 		}
-		if ok, _ := cmd.Flags().GetBool(tracingTlS); !ok {
+		if ok, err := cmd.Flags().GetBool(tracingTlS); err != nil {
```

Comment:
`metricsTls`

Reply:
Oh, good catch. Thank you!!!

Reply:
Copy of https://github.com/celestiaorg/celestia-node/pull/907#discussion_r935371791